components/hint-text/src/index.js

```diff
@@ -13,7 +12,7 @@ const StyledHint = styled('span')(
     display: 'block',
     // NB non-responsive marginBottom here
     marginBottom: SPACING_POINTS[3],
-    color: `${SECONDARY_TEXT_COLOUR}`,
+    color: '#505a5f',
```

Comment:
This isn't ideal because really 'SECONDARY_TEXT_COLOUR' should be this value anyway in govuk-colours to align with https://design-system.service.gov.uk/styles/colour/.
However, govuk-colours is out of date, so this is a hack which might have to do for now.

It might be worth trying to raise a PR on the govuk-colours repo directly instead though, and see if we can get it merged and published officially?
https://github.com/penx/govuk-colours/pulls?q=is%3Apr+is%3Aclosed

Reply:
Yeah, isn't ideal.
Will do a PR directly to the govuk-colours repo and see what happens

Reply:
@david-kasali @paul-burnett we should probably coordinate at some point as I'm doing some work on govuk-react and it would be good to include fixes for any issues you have had.